### DISCUSSION TITLE
Add vehicle in source

### DISCUSSION THREAD

**Discussion by BEDAVEURDev** [MAFINS/MenyooSP#595] [General]
Hello, 

I've download source, but i didn't really know how can i add vehicle for example : 

			{ VEHICLE_CRUISER, "pandm_default", "cruiser" },
			
If i want add : 
		
		      { VEHICLE_INDUCTOR, "What need to put here ?", "inductor" },
I know how to compil it.
I hope someone can help me.

**Comment by metoxys** [comment]
What are you trying to add where?

**Reply by BEDAVEURDev** [reply]
Hello i want to add vehicle of new DLC, thank for answer.

**Reply by metoxys** [reply]
Where and why are you trying to add them? Are you trying to add them to the list of the vehicle spawner?

**Reply by BEDAVEURDev** [reply]
Yep i want to add it in vehicle spawner ! i want to add cuz are not in atm.

**Reply by metoxys** [reply]
No need to edit or compile the source for that

Grab VehicleList.xml from #581 and replace the one you have in your /menyooStuff/

**Reply by BEDAVEURDev** [reply]
Idk why, but i've add in my menyooStuff folder and didn't have car in my menyoo :/

**Reply by metoxys** [reply]
Vehicle spawner, scroll all the way down to manually write the model name, write something like inductor or stingertt, see if the vehicle spawns or you get an error message

**Reply by BEDAVEURDev** [reply]
got error message :/

**Reply by metoxys** [reply]
Then fix your game first

Chances are your dlclist is messed up (old update.rpf)

**Reply by BEDAVEURDev** [reply]
Okay i'll going trying that
